<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmAsignarOpciones</class>
 <widget class="QWidget" name="FrmAsignarOpciones">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>577</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Asignar Opciones</string>
  </property>
  <widget class="QGroupBox" name="gb_datosAsignarPermisos">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>731</width>
     <height>531</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Asignación de Opciones</string>
   </property>
   <widget class="QComboBox" name="cbx_rol">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>291</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_rol">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Rol</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_opcion">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Opción</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbx_opcion">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>291</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QTableWidget" name="tw_registrosRolOpcion">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>661</width>
      <height>241</height>
     </rect>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>264</number>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <column>
     <property name="text">
      <string>id_RolOpcion</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>id_Rol</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>id_Opcion</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Rol</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Opcion</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btn_asignar_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>80</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:400;&quot;&gt;Asigna un rol a un usuario&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
background-color:rgb(7, 125, 8)</string>
    </property>
    <property name="text">
     <string>Asignar</string>
    </property>
    <property name="icon">
     <iconset resource="../recursos/iconosApp.qrc">
      <normaloff>:/iconosCrud/add40.png</normaloff>:/iconosCrud/add40.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_eliminar_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>130</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:400;&quot;&gt;Elimina un rol que este asignado a un usuario&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
background-color:rgb(201, 0, 0)</string>
    </property>
    <property name="text">
     <string>Eliminar</string>
    </property>
    <property name="icon">
     <iconset resource="../recursos/iconosApp.qrc">
      <normaloff>:/iconosCrud/delete40.png</normaloff>:/iconosCrud/delete40.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_buscador_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_buscar_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>200</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Buscar</string>
    </property>
    <property name="icon">
     <iconset resource="../recursos/iconosApp.qrc">
      <normaloff>:/iconosCrud/search40.png</normaloff>:/iconosCrud/search40.png</iconset>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>cbx_rol</tabstop>
  <tabstop>cbx_opcion</tabstop>
 </tabstops>
 <resources>
  <include location="../recursos/iconosApp.qrc"/>
 </resources>
 <connections/>
</ui>
